packages/jest-haste-map/src/lib/extract_requires.js

```diff
@@ -12,8 +12,8 @@ const blockCommentRe = /\/\*[^]*?\*\//g;
 const lineCommentRe = /\/\/.*/g;
 
 const replacePatterns = {
-  EXPORT_RE: /(\bexport\s+(?:[^'"]+\s+from\s+)??)(['"])([^'"]+)(\2)/g,
```

Comment:
Are there any tests that cover this currently? I didn't see anything and I'm not sure how it is used so I wasn't able to add a test for this.